### PR TITLE
RE_Fix

### PR DESCRIPTION
**Pull request by Lusarien**
Hey guys in #527 Dom pointed out that regex was missing some of the things that it shouldn't have. I looked into it a little bit and it seems this is a pretty solid regex for validating e-mails, thoughts comments concerns would be appreciated, thanks :)

Reference:
http://emailregex.com/

### CODE REVIEW

**Comment by thaeli** [comment]
Regex is a inherently subpar way to validate emails. This is a more comprehensive approach: https://github.com/mailgun/flanker

File: development-defaults.ini [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 path = "/uber"
 hostname = "localhost"
-url_root = "https://localhost:4443"
+url_root = "https://localhost"
```
Comment (kitsuta) [inline development-defaults.ini]: Was there a particular reason to do this, or was it just for your system?

You can use or create a `development.ini` file in the same directory as this file - it lets you override these config options without constantly having to check them into the repo.

**Comment by binary1230** [comment]
Hey don't merge this PR, you should close it and re open it not against master but against insert_admin_via_web_ui

File: development-defaults.ini (hunk 1 above)
Comment (binary1230) [inline development-defaults.ini]: ignore, PR was against the wrong branch, this is docker test stuff

**Pull request closed without merging**